Certified Sail Extension (CSE) Examples/cse/mapField/v1/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel='stylesheet' href='./map.css'></link>
+</head>
+
+<body>
+  <div id='map-container'></div>
+  <script src='APPIAN_JS_SDK_URI'></script>
+  <script>
+    var mapState = {};
+    var newKey, newLocation, currentKey;
+    var scriptId = 'scriptTag';
+
+    // Handle a new user input location from SAIL.
+    Appian.Component.onNewValue(function (newValues) {
+      newKey = newValues.key;
+      newLocation = newValues.location;
+
+      if (newKey && currentKey !== newKey) {
+        var oldScript = document.getElementById(scriptId);
+        if (oldScript) {
+          oldScript.parentElement.removeChild(oldScript);
+          document.getElementById('map-container').innerHTML = ''
+          google = null;
+          mapState = {};
+        }
+
+        currentKey = newKey;
+        var s = document.createElement('script');
+        s.setAttribute('id', scriptId)
+        s.setAttribute('src', 'https://maps.googleapis.com/maps/api/js?key=' + newKey + '&callback=initMap');
+        document.body.appendChild(s);
+      }
+      if (mapState.scriptLoaded && currentLocation !== newLocation) {
+        changeLocation(newLocation);
+      }
+    });
+
+    function initMap() {
+      mapState.scriptLoaded = true;
+      changeLocation(newLocation)
+    }
+
+    function changeLocation(newLocation) {
+      currentLocation = newLocation;
+      // Obtain coordinates for user input location
+      mapState.geocoder = mapState.geocoder || new google.maps.Geocoder();
+      mapState.geocoder.geocode({ 'address': newLocation }, function (results, status) {
+        var latLng = results[0].geometry.location;
+        if (mapState.map) {
+          mapState.map.setCenter(latLng);
+        } else {
+          renderMap(latLng);
+        }
+      });
+    }
+
+    function renderMap(latLng) {
+      // Render a map at the requested coordinates
+      mapState.container = document.getElementById('map-container');
+      mapState.map = new google.maps.Map(mapState.container, {
+        zoom: 12,
+        center: latLng,
+        mapTypeId: google.maps.MapTypeId.ROADMAP
+      });
+
+      mapState.map.addListener('click', function (event) {
+        if (mapState.pin) {
+          mapState.pin.setMap(null); // Remove previous pin
```

Comment:
This comment was previously put on the same line to save space for an example since the file was shorter - we should probably move it to the line above for consistency